src/app.js

```diff
@@ -46,6 +46,17 @@ app.use(passport.session());
 
 app.use('/auth', auth.router);
 
+// match /cm, /cm/, /cm/new
+// but not /cmgame/, /cmgame/session
```

Comment:
Why not these specifically?

Reply:
In case we will have eventually a game, which name starts with `cm`. so as regex "says":

- path starts with `/cm`
- then followed by `/` (e.g. `/cm/`) OR `end_of_string` (e.g. `/cm`)
- and then 0+ characters (e.g. `/cm/#/puzzles/new`)

So the problem could be with "starts with `/cm`" as game name may also start with it. But this particular regex handles all correctly.